libs/libs-pomfix/bin/pom-fix

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env node
+
+var pkg = require('../package.json')
+var es = require('event-stream')
+var util = require('../lib/util')
+var chalk = require('chalk')
+
```

Comment:
Nit: I'd try to keep things consistent and either put blank lines between all `requires`, or none.